app/assets/stylesheets/components/_user_profile_layout.sass

```diff
@@ -0,0 +1,27 @@
+$UserProfileLayout-body-spacing: $Theme-spacing-base
+$UserProfileLayout-color-title: Theme-color(text, title-page)
+$UserProfileLayout-fontSize-title: Theme-fontSize(title, large, font-size)
+$UserProfileLayout-fontWeight-title: Theme-fontWeight(bold)
+$UserProfileLayout-spacing-title: 4 * $UserProfileLayout-body-spacing
+
+.UserProfileLayout-header
```

Comment:
Also, as discussed, since you'll need a 3 column layout, maybe this component doesn't make sense by itself.
Maybe you could reuse the Grid component